components/list/README.md

```diff
@@ -158,6 +158,12 @@ The `d2l-list` is the container to create a styled list of items using `d2l-list
 
 The `d2l-list` supports selectable items within a list, including both single and multi selection. Selection is enabled when `d2l-list-item`s have the `selectable` attribute. When items are selectable, multiple selection is the default behaviour, however the `selection-single` attribute can be applied to the `d2l-list` to enable single selection. A `d2l-list-header` component can be added to `d2l-list`'s `header` slot to provide select-all and bulk actions.
 
+## Pageable Lists
```

Comment:
Ignore this - not sure why this got pulled in. Probably something to do with the conflict resolution when I updated this old branch.